internal/provider/resource_cloudflare_access_group_test.go

```diff
@@ -394,6 +394,62 @@ resource "cloudflare_access_group" "%[2]s" {
 }`, accountID, rnd, githubOrg, team)
 }
 
+func testAccCloudflareAccessGroupConfigIPList(resourceName string, ip string, identifier AccessIdentifier) string {
+	return fmt.Sprintf(`
+resource "cloudflare_access_group" "%[1]s" {
+  %[3]s_id = "%[4]s"
+  name     = "%[1]s"
+
+  include {
+    ip_list = ["%[2]s"]
+  }
+}`, resourceName, ip, identifier.Type, identifier.Value)
+}
+
+func TestAccCloudflareAccessGroupConfig_IPlistInclude(t *testing.T) {
+	rnd := generateRandomResourceName()
+	name := fmt.Sprintf("cloudflare_access_group.%s", rnd)
+	ip := "1.1.1.2"
+	resource.Test(t, resource.TestCase{
```

Comment:
you'll want to split these into two different test scenarios. overlapping them like this is likely to cause testing issues.